docs/index.rst

```diff
@@ -115,7 +115,7 @@ Searching for nanopublications
 
     # Search for all nanopublications containing the text 'fair'
     results = client.find_nanopubs_with_text('fair')
-    print(results)
+    print(list(results))
```

Comment:
Should we note here that only the first 10000 results will appear in the resultant list? (assuming that is the pagination limit?)

Reply:
The pagination limit is 1000, but the list will be as large as the number of matching nanopublications (it just eats up the entire generator). But I will change this to:
```python
for result in results:
    print(result)
```